docs/querying/api.md

```diff
@@ -696,6 +696,66 @@ $ curl http://localhost:9090/api/v1/status/flags
 }
 ```
 
+### Runtime Information
+
+The following endpoint returns various runtime information properties about the Prometheus server:
+
+```
+GET /api/v1/status/runtimeinfo
+```
+
+The returned values are of different types, depending on the nature of the runtime property.
+
+```json
+$ curl http://localhost:9090/api/v1/status/runtimeinfo
+{
+  "status": "success",
+  "data": {
+    "startTime": "2019-11-02T17:23:59.301361365+01:00",
+    "CWD": "/",
+    "reloadConfigSuccess": true,
+    "lastConfigTime": "2019-11-02T17:23:59+01:00",
+    "chunkCount": 873,
+    "timeSeriesCount": 873,
+    "corruptionCount": 0,
+    "goroutineCount": 48,
+    "GOMAXPROCS": 4,
+    "GOGC": "",
+    "GODEBUG": "",
+    "storageRetention": "15d"
+  }
+}
+```
+
+**NOTE**: The exact returned runtime properties may change without notice between Prometheus versions!
```

Comment:
I don't think we need the exclamation mark.

Reply:
I thought the exclamation mark would keep the API stability lawyers away ;) Replaced with a period.

Reply:
We need to mention it, not shout it from the rooftops :)